backend/services/prisonerProfileService.ts

```diff
@@ -196,6 +197,17 @@ export default ({
     const canCalculateReleaseDate =
       userRoles && (userRoles as any).some((role) => role.roleCode === 'RELEASE_DATES_CALCULATOR')
 
+    const getPrisonerSearchDetails = async () => {
+      const response = await offenderSearchApi.getPrisonersDetails(systemContext, [offenderNo])
+      const prisonerSearchDetails = response && response[0]
+      return {
+        hospital: prisonerSearchDetails?.dischargedHospitalDescription,
+        isRestrictedPatient: prisonerSearchDetails?.restrictedPatient,
+        indeterminateSentence: prisonerSearchDetails?.indeterminateSentence,
+      }
+    }
+    const { hospital, isRestrictedPatient, indeterminateSentence } = await getPrisonerSearchDetails()
+
```

Comment:
Rather than calling prisoner offender search in quick look / sentence and release dates, call it a single time here so we can also grab some restricted patient details.